webpack/assets/javascripts/react_app/components/notifications/NotificationDrawer.js

```diff
@@ -24,6 +24,7 @@ class NotificationDrawer extends Component {
 
   onChange(actionType) {
     switch (actionType) {
+      case ACTIONS.MARK_AS_READ:
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({ notifications: NotificationsStore.getNotifications() });
```

Comment:
I presume you mean the redux conversion. It will be good to get rid of the `switch` statements. Although formally not code smells, they make me uncomfortable.

Odd that linting didn't pick up the error since we have `"no-fallthrough": "error"` specified